README.md

```diff
@@ -1,46 +1,143 @@
+* Each app managed by `kubernetes-deploy` must have its own exclusive Kubernetes namespace.**
+
+
+**This requirement can be bypassed with the `--no-prune` option, but it is not recommended.
+
+## Installation
+
+1. [Install kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl) (requires v1.6.0 or higher) and make sure it is available in your $PATH
+2. Set up your [kubeconfig file](https://kubernetes.io/docs/tasks/access-application-cluster/authenticate-across-clusters-kubeconfig/) for access to your cluster(s).
+3. `gem install kubernetes-deploy`
 
-    $ bundle
 
-Or install it yourself as:
 
-    $ gem install kubernetes-deploy
 
 ## Usage
 
-`kubernetes-deploy <app's namespace> <kube context> --template-dir=DIR`
+`kubernetes-deploy <app's namespace> <kube context>`
 
-Requirements:
+*Environment variables:*
 
- - kubectl 1.6.0+ binary must be available in your path
- - `ENV['KUBECONFIG']` must point to a valid kubeconfig file that includes the context you want to deploy to
- - The target namespace must already exist in the target context
- - `ENV['GOOGLE_APPLICATION_CREDENTIALS']` must point to the credentials for an authenticated service account if your user's auth provider is GCP
- - `ENV['ENVIRONMENT']` must be set to use the default template path (`config/deploy/$ENVIRONMENT`) in the absence of the `--template-dir=DIR` option
+- `$REVISION` **(required)**: the SHA of the commit you are deploying. Will be exposed to your ERB templates as `current_sha`.
+- `$KUBECONFIG`  **(required)**: points to a valid kubeconfig file that includes the context you want to deploy to
+- `$ENVIRONMENT`: used to set the deploy directory to `config/deploy/$ENVIRONMENT`. You can use the `--template-dir=DIR` option instead if you prefer (**one or the other is required**).
+- `$GOOGLE_APPLICATION_CREDENTIALS`: points to the credentials for an authenticated service account (required if your kubeconfig `user`'s auth provider is GCP)
 
-The tool also provides a task for restarting all of the pods in one or more deployments.
-It triggers the restart by touching the `RESTARTED_AT` environment variable in the deployment's podSpec.
-The rollout strategy defined for each deployment will be respected by the restart.
 
-The following command will restart all pods in the `web` and `jobs` deployments:
+*Options:*
+
+- `--bindings=BINDINGS`: Makes additional variables available to your ERB templates. For example, `kubernetes-deploy my-app cluster1 --bindings=color=blue,size=large` will expose `color` and `size`.
+- `--verbose-log-prefix`: Prefixes each log line with `[#{context}][#{namespace}][#{severity}][#{datetime}]` instead of just `[#{severity}][#{datetime}]`. Useful if you're running multiple concurrent deploys in Shipit.
```

Comment:
(because IMO things like `--verbose-log-prefix` are not going to be used frequently enough to highlight in the README)